libs/partners/nvidia-ai-endpoints/langchain_nvidia_ai_endpoints/chat_models.py

```diff
@@ -116,6 +117,23 @@ class ChatNVIDIA(nvidia_ai_endpoints._NVIDIAClient, SimpleChatModel):
             response = model.invoke("Hello")
     """
 
+    model_kws: List[str] = [
```

Comment:
delete

Reply:
Is necessary

Reply:
For what?

Reply:
https://catalog.ngc.nvidia.com/ai-foundation-models
[Mixtral_8x7b is pretty representative](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/ai-foundation/models/mixtral-8x7b/documentation)

Reply:
I'll yield. Your option can work for now. Thanks :D